openapi/dali-maxrec.yaml

```diff
@@ -0,0 +1,6 @@
+name: MAXREC
+in: query
+description: request a specific limit on number of rows to return
+schema:
+    type: integer
+    format: int64
```

Comment:
It might be worth defining the schema with the limit `minimum: 0` to be explicit about the range. Minimum is [inclusive for OpenAPI 3](https://swagger.io/docs/specification/v3_0/data-models/data-types/#numbers) so that would cover `MAXREC=0` for TAP.

Reply:
I explicit set `exclusiveMinimum: false` as well just to be clear.

Reply:
I like it better, actually. I'm sure it'll save someone the effort of looking it up later.